docs/support/tutorials/index.md

```diff
@@ -72,6 +72,9 @@
 
 You can find Pouta tutorials in [Cloud Services > Pouta > Tutorials](../../cloud/pouta/tutorials/index.md)
 
+## Pukki - database cloud
+* [How to migrate a database to Pukki (DBaaS)](./pukki_data_migration.md)
```

Comment:
Maybe update the name to `How to migrate a PostgreSQL database to Pukki`

Reply:
done